### DISCUSSION TITLE
Can you create example app using getcandy package from zero?

### DISCUSSION THREAD

**Discussion by devmansurov** [lunarphp/lunar#80] [Q&A] [answered]
Many novice developers may not know how to use your platform in practice. There is of course documentation for this but if you build a normal base application using your platform with as much functionality as possible that exists in your repository it would be easier for everyone to learn your platform. For example, from creating categories, attributes, options and products to placing an order. It would also be nice if you separate the frontend from the backend using the REST API architecture. It will also help for the development of your product. Thanks for the development. Good luck to you all.

**Comment by glennjacobs** [comment, marked as answer]
Yep, that is all planned and will be coming in the following weeks :-)
